hybrid-standalone/modules/ROOT/pages/managing-properties-hybrid.adoc

```diff
@@ -0,0 +1,117 @@
+= Managing Properties
+
+Manage application properties for your Mule applications deployed to hybrid standalone instances through centralized configuration management, enabling dynamic property updates, environment-specific settings, and secure property handling across your infrastructure.
+
+== Overview
+
+Runtime Manager provides comprehensive property management capabilities for applications deployed to hybrid standalone instances, enabling you to:
+
+* Define and manage application properties centrally through Runtime Manager
+* Override application-bundled properties without redeployment
+* Configure environment-specific settings for different deployment stages
+* Secure sensitive configuration data through encrypted property storage
+* Manage properties across individual servers, server groups, and clusters
+
+**Property Management Approaches**
+
+* **Properties Files**: Include property placeholder YAML files in application deployment archives
+* **Runtime Manager Properties**: Override and manage properties through the Runtime Manager console
+* **System Variables**: Configure properties at deployment time using Java system environment variables
+* **External Configuration**: Use external property sources for advanced configuration management
+
+**Property Precedence**
+
+Properties defined in Runtime Manager take precedence over application-bundled properties, enabling environment-specific configuration without modifying application code.
+
+== Property Security
+
+Runtime Manager implements comprehensive security measures for property management:
+
+* **Transmission Security**: All property values are encrypted over HTTPS during transmission between Runtime Manager and Mule runtime
+* **Storage Security**: Properties are encrypted when stored in the Runtime Manager internal database
+* **Access Control**: Property values are displayed only through the Runtime Manager Settings page with appropriate permissions
+* **Audit Trail**: Property modifications are tracked for compliance and security auditing
+
```

Comment:
For lines 30 to 33:
Same as above, we should use bold only for UI elements. Either use a definitions list or remove the bold internal headings.
Also,  the list items should take final punctuation (periods) because they are constructed as complete sentences.